pkg/releaseinfo/releaseinfo_test.go

```diff
@@ -58,6 +58,7 @@ func TestReleaseInfo_GetReleaseData(t *testing.T) {
 			releaseVersion:            "1.0.1",
 			expectedK8sVersion:        "15.0.1",
 			expectedK8sVersionEOLDate: "2020-10-20",
+			expectedIsK8sVersionEOL:   true,
```

Comment:
This is something that I missed in a previous PR, and it made a date-specific test fail